### ISSUE TITLE
Support for $main and $aux built-ins in boundary constraints

### ISSUE THREAD

**Issue by Overcastan** [0xPolygonMiden/air-script#114]
Related issue: #26 
Implement column access by index as it described in related issue for boundary constraints.

**Comment by grjte** [comment]
I think we may want to look at this after #106

**Comment by bobbinth** [comment]
Closing this as no longer relevant.
